CvGameCoreDLL_Expansion2/Lua/CvLuaUnit.cpp

```diff
@@ -1811,23 +1811,6 @@ int CvLuaUnit::lIsRangeAttackOnlyInDomain(lua_State* L)
 	return 1;
 }
 
-//------------------------------------------------------------------------------
-int CvLuaUnit::lIsCoastalAttackOnly(lua_State* L)
```

Comment:
It's not possible to build the dll after you removed this (forgot to update line 338). Looks like there's a reference to this in Bombardment.lua as well.

Reply:
It shouldn't affect build, only having error when Bombardment.lua is run (when a human selects a city to attack, which I didn't test).

Reply:
fixed in latest commit